pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -105,6 +105,10 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation, operationType
 			Name: "Syncing shoot cluster information to seed",
 			Fn:   flow.TaskFn(botanist.SyncClusterResourceToSeed).RetryUntilTimeout(defaultInterval, defaultTimeout),
 		})
+		_ = g.Add(flow.Task{
```

Comment:
Just to double check: Is it correct that no other task is depending on this one?